src/Library/Lod/Internal/LodDirectory.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include "Library/Lod/Internal/LodFile.h"
+
+
+struct LodDirectory final {
+    std::string name;
+    std::vector<LodFile> files;
+
+    size_t file_headers_offset = 0;
```

Comment:
Code style: fileHeaderOffset

Reply:
it's mechanical
sorry, will pay more attention